tempus_fugit_minecraft/main.py

```diff
@@ -423,20 +422,15 @@ def process_entire_queue(self):
             self._dequeue()
 
     @staticmethod
-    def generate_clouds(n, num_of_clouds=250):
+    def generate_clouds_positions(world_size, num_of_clouds=250):
         """
         Generate the position of the clouds on the sky.
-
-        Inputs: n = 1/2 size of the world.
-                num_of_clouds = default clouds to be generated =250
-
-        Output: return a list of lists; each inner list represents a set of cloud blocks.
         """
-        o = n - 10
+        game_margin = world_size
```

Comment:
More understandable with these variables, I agree with Mahmoud's note about making variables small letters and capital letters, depending on changing values.